tuner/bayes_opt_gp/setup.cfg

```diff
@@ -0,0 +1,10 @@
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+install_requires =
+    dffml>=0.4.0
+    bayesian-optimization>=1.2.0
```

Comment:
is there an implementation in a library we already have in requirements? say sklearn? We always wan't to have minimal dependencies

Reply:
There is BayesSearchCV in the scikit-optimize library (which is also a separate depedency). But it requires that the model implement the sklearn estimator API.